src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -47,13 +48,22 @@ class OppgaveService(
     fun opprettOppgave(
         behandlingId: UUID,
         oppgavetype: Oppgavetype,
+        vurderHenvendelseOppgavetype: VurderHenvendelseOppgavetype? = null,
         tilordnetNavIdent: String? = null,
         beskrivelse: String? = null,
         mappeId: Long? = null, // Dersom denne er satt vil vi ikke prøve å finne mappe basert på oppgavens innhold
         prioritet: OppgavePrioritet = OppgavePrioritet.NORM,
     ): Long {
         val oppgaveFinnesFraFør =
-            oppgaveRepository.findByBehandlingIdAndTypeAndErFerdigstiltIsFalse(behandlingId, oppgavetype)
+            if (oppgavetype == Oppgavetype.VurderHenvendelse && vurderHenvendelseOppgavetype != null) {
+                oppgaveRepository.findByBehandlingIdAndTypeAndVurderHenvendelseOppgavetype(
+                    behandlingId,
+                    oppgavetype,
+                    vurderHenvendelseOppgavetype,
+                )
+            } else {
+                oppgaveRepository.findByBehandlingIdAndTypeAndErFerdigstiltIsFalse(behandlingId, oppgavetype)
+            }
```

Comment:
Kan vi flytte ut denne til en egen hjelpemetode for bedre lesbarhet?

Reply:
Enig, flyttet til egen metode